elements/pfe-switch/src/pfe-switch.scss

```diff
@@ -0,0 +1,219 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: switch;
+
+// PFE Config
+$font-size:     16px;
+$line-height:   24px;
+
+// Setting all toggle sizing, will be converted to em in styles
+$toggle-width:        42px;
+$toggle-height:       $line-height;
+$toggle-knob-diameter:   14px;
+$toggle-knob-offset:  5px;
+
+///
+/// Convert px to em
+///
+/// @param $value {length}  Value to convert with px suffix
+/// @param $base-size {length} Optional, font size to use to calculate em
+/// @return {length} Em version of input
+@function px-to-em($value) {
```

Comment:
This is a great addition to pfe-sass functions in the `tools` file

Reply:
Would this function do what you need?

```
// Return a pixel size in ems
@function pfe-size-pem($pxval, $base: $pfe-global--font-size-root) {
    @return pfe-strip-unit($pxval) / $base * 1em;
}
```

Reply:
It's in the `elements/pfe-sass/functions/_1-tools.scss`

Reply:
It is! thanks!